tests/core/test_flow.py

```diff
@@ -2960,13 +2978,18 @@ def slow_fn():
         assert not os.path.exists(FILE)
 
         start_time = time.time()
-        state = flow.run(executor=executor)
+        with assert_daemon_warning if daemon_process else nullcontext():
```

Comment:
This is failing to capture the warning -- cannot capture it using `capfd` either. It's being logged though. Hmm.

Reply:
Resolved by adding it to the Timeout error message